httpapi_plugins/ftd.py

```diff
@@ -129,6 +129,31 @@ def refresh_token_payload(refresh_token):
             raise ConnectionError(
                 'Server returned response without token info during connection authentication: %s' % response)
 
+    def _lookup_login_url(self, payload):
+        """When token generation URL is not defined in the inventory we may try to find it on the fly.
+        :param payload: Token request payload
+        :type payload: dict
+        :return: token generation response
+        """
+        for url in self._get_known_token_paths():
+            try:
+                response = self._login_api_call(payload, url)
+            except ConnectionError:
+                display.vvvv('REST:request to {0} failed because of connection error'.format(url))
```

Comment:
Please, reuse `_display` helper method for logging.

Reply:
_display - has several predefined params which we can't control here but they are still required.

Reply:
Ok, it makes sense.